### ISSUE TITLE
us-east-2 error

### ISSUE THREAD

**Issue by mluypaert** [awslabs/ec2-migrate-longer-id#3]
When running this script (with the `--status` option) the script run fine, until it comes at us-east-2 (your newly launched region), at with point it crashes.

The eror message is this:
>('error code:', 400, '\nrequest ID:', '9c54716d-637a-4d4d-8767-5b6d0c74efc3')
>('sleep for:', 0.2)
>Traceback (most recent call last):
>  File "./migratelongerids.py", line 394, in <module>
>    main()
>  File "./migratelongerids.py", line 379, in main
>    call_status(regions, usersandids, idresources)
>  File "./migratelongerids.py", line 87, in call_status
>    describeid(k, resource_number, resources)
>  File "./migratelongerids.py", line 253, in describeid
>    for line in id_client_response['Statuses']:
>UnboundLocalError: local variable 'id_client_response' referenced before assignment

Could you please fix this (probably just output True for all iam entries in this region, or just ignore this region?)

**Comment by brete** [comment]
Hi,

I was able to reproduce the issue on my end. I did some quick digging and this is not a problem with the application, but with the APIs. As I work with the team that is responsible for this API call, I'm going to reach out to them and see if we can get this corrected.

I'm going to leave this open for now. Although, this does bring up a good opportunity to add some additional functionality/robustness for the application in the event that the call fails for a region to continue functioning.
